python/cudf/cudf/core/index.py

```diff
@@ -2357,6 +2357,23 @@ def is_leap_year(self):
         res = is_leap_year(self._values).fillna(False)
         return cupy.asarray(res)
 
+    @property
+    def quarter(self):
+        """
+        Integer indicator for which quarter of the year the date belongs in.
+
+        There are 4 quarters in a year. With the first quarter being from
+        January - March, second quarter being April - June, third quarter
+        being July - September and fourth quarter being October - December.
+
+        Returns
+        -------
+        Series
+        Integer indicating which quarter the date belongs to.
+        """
+        res = extract_quarter(self._values).fillna(False)
```

Comment:
An integer value of 0 indicates Q0, i.e. Jan-Mar, correct? Doesn't filling nulls with `False` result in the equivalent of integer 0, which means that null values would get binned into Q0?

Reply:
There is no such thing as Q0, it starts at Q1 so I don't think this would be an issue.

Reply:
What happens in this scenario?

```python
import pandas as pd
​
s = pd.Series(["2021-01-01", None], dtype="datetime64[ms]")
s.dt.quarter
0    1.0
1    NaN
dtype: float64
```

Reply:
It originally returned 0, but after removing the `.fillna(False)`, it replaced the 0 with `cudf.NA`, which I think is the correct behaviour?

Reply:
That LGTM

Reply:
Yeah that was more my point, 0 shouldn't be the default output whether quarters are 0- or 1-indexed.